mod/deckorate_datasets/set/type/dataset.rb

```diff
@@ -9,14 +10,38 @@
 card_reader :data_subset, type: :search_type
 card_reader :wikirate_topic, type: :list
 
-def parent_dataset_card
-  Card[parent_dataset]
+format :html do
+  def breadcrumb_items
+    breadcrumb_array = [
+      link_to("Home", href: "/"),
+      link_to_card(:dataset, "Datasets"),
+      render_name
+    ]
```

Comment:
you could just say `breadcrumb_array = super`, right? Less duplication, more robust if the original method is edited?